pkg/dbsyncers/policy_syncer.go

```diff
@@ -149,10 +151,87 @@ func updateComplianceStatus(ctx context.Context, k8sClient client.StatusClient,
 		policy.Status.ComplianceState = policiesv1.Compliant
 	}
 
-	err := k8sClient.Status().Patch(ctx, policy, client.MergeFrom(originalPolicy))
+	placements, err := getPlacements(k8sClient, policy)
+	if err != nil {
+		return err
+	}
+	policy.Status.Placement = placements
+
+	err = k8sClient.Status().Patch(ctx, policy, client.MergeFrom(originalPolicy))
 	if err != nil && !errors.IsNotFound(err) {
 		return fmt.Errorf("failed to update policy CR: %w", err)
 	}
 
 	return nil
 }
+
+func getPlacements(k8sClient client.Client, instance *policiesv1.Policy) ([]*policiesv1.Placement, error) {
+	log.Info("get the placements for the policy", "policy", instance.GetName())
+	pbList, err := getPlacementBindingList(k8sClient, instance)
+	if err != nil {
+		return nil, err
+	}
+	var placements []*policiesv1.Placement
+	for _, pb := range pbList.Items {
+		subjects := pb.Subjects
+		for _, subject := range subjects {
+			if !(subject.APIGroup == policiesv1.SchemeGroupVersion.Group &&
+				subject.Kind == policiesv1.Kind &&
+				subject.Name == instance.GetName()) {
+				continue
+			}
+			p, err := getPlacementsPerBinding(k8sClient, pb, instance)
```

Comment:
can you use longer and more descriptive names? we have now variables `p`, `pb`, `plr`, `pbList`...
this code gets very confusing with these one/two char names and gets hard to follow.

can we call the variables with a name that describes them?
e.g. `placementRule`, `placementBinding`, `placementBindingList`, etc.